modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc

```diff
@@ -80,7 +81,7 @@ If you chose not to create your custom policy project using the provided Maven a
 </repository>
 --
 +
-. Set up your Exchange server as a distribution management:
+* Set up your Exchange server as distribution management:
```

Comment:
"server as management" or "server as manager" (or "server to use"?)

Reply:
Let me think about this one. The section in the file is actually distributionManagement.